src/Atoms/Inputs/Password.js

```diff
@@ -43,25 +43,32 @@ class Password extends React.Component<PropsType, StateType> {
   }
 
   handleSwitchClick = () => {
-    if (this.state.type === 'password') {
-      this.setState({ type: 'text' })
-    } else {
-      this.setState({ type: 'password' })
-    }
+    this.state.type === 'password'
+      ? this.setState({ type: 'text' })
+      : this.setState({ type: 'password' })
+  }
+
+  renderToggle = () => {
+    const { disabled } = this.props
```

Comment:
Don't use `disabled` from `this.props`, you should get it from the parameters. That way that function is pure.

```javascript
renderToggle = (__, disabled)
```